components/interface/VFBGraph/VFBGraph.js

```diff
@@ -184,16 +186,58 @@ class VFBGraph extends Component {
       if (event.isComposing || event.keyCode === 16) {
         self.shiftOn = false;
       }
-    });    
+    });
   }
 
   componentDidUpdate () {
     let self = this;
+
```

Comment:
@jrmartin I added this to handle the problem where I was opening the component before the model and template were loaded. 
The whole problem sit in the state constructor [here](https://github.com/VirtualFlyBrain/geppetto-vfb/blob/c486208375941e6655761df3c8d0010f36f482f8/components/interface/VFBGraph/VFBGraph.js#L127) since, if I open the component before model and template are loaded, the focus instance is still empty and that will be to.
To fix this I am checking the currentQuery string so that if this has not been initialised I try to use either the props.instance or the props.instanceOnFocus (you can see with the debugger why) and getting the this.firstLoad initialised to false to stop this loop. 
Could you please review and in case change this if you don't agree with this change?
Thx